content/ru/_index.html

```diff
@@ -0,0 +1,62 @@
+---
+title: "Первоклассная оркестрация контейнеров"
+abstract: "Автоматизированное развёртывание, масштабирование и управление контейнерами."
+cid: home
+---
+
+{{< deprecationwarning >}}
+
+{{< blocks/section id="oceanNodes" >}}
+{{% blocks/feature image="flower" %}}
+### [Kubernetes (K8s)]({{< relref "/docs/concepts/overview/what-is-kubernetes" >}}) - это открытое программное обеспечение для автоматизации развёртывания, масштабирования и управления контейнеризированными приложениями.
```

Comment:
My guess is that netlify is failing because of the shortcode in this line.

Reply:
Ready to review 👀